core/src/main/java/fr/sncf/osrd/infra/api/tracks/undirected/SpeedLimits.java

```diff
@@ -19,14 +19,14 @@ public SpeedLimits(double defaultSpeedLimit, ImmutableMap<String, Double> speedL
 
     /** Directly creates a SpeedLimits instance from the railjson dict */
     public static SpeedLimits from(RJSSpeedSection rjsSpeedSection) {
+        assert !Double.isNaN(rjsSpeedSection.speedLimit);
```

Comment:
`rjsSpeedSection.speedLimit` can be `NaN`, it's optional in the RJS schema. It can be used to add limits for specific tags without affecting the rest.

Reply:
can then I should probably make it default to +inf